lib/matplotlib/cbook.py

```diff
@@ -2244,6 +2244,21 @@ def _reshape_2D(X):
     return X
 
 
+def ensure_3d(arr):
```

Comment:
Do we want to make this a public function? We now have `_check_1d()`, `_reshape_2D`, and this new `ensure_3d` (notice the inconsistency with the case of "d/D").

Why don't we make this private, and make a new issue to unify the logic across these three functions later (if possible)?